src/test/lib/lit-extended_test.ts

```diff
@@ -51,6 +51,35 @@ suite('lit-extended', () => {
       assert.equal(container.innerHTML, '<div foo="1bar2baz3"></div>');
     });
 
+    test('renders a boolean attribute as an empty string when truthy', () => {
+      let t = (value: any) => html`<div foo?="${value}"></div>`;
+
+      render(t(true), container);
+      assert.equal(container.innerHTML, '<div foo=""></div>');
```

Comment:
in HTML <div foo=""> is the same as <div foo> or <div foo=''>.

Wouldn't going for <div foo> seem more logical?

Reply:
Even if I set `<div foo="">` manually in devtools, it turns into `<div foo>`

Reply:
I think this is the only way to do it from DOM APIs.